filebrowser_safe/base.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, path):
     def name(self):
         return self.path
 
-    @property
+    @cached_property
```

Comment:
An attempt to avoid one more call. But didn't help. Didn't feel like it would be bad to leave it.